test/libsolidity/smtCheckerTests/operators/conditional_assignment_5.sol

```diff
@@ -24,4 +24,4 @@ contract C {
 }
 // ----
 // Warning 2072: (282-288): Unused local variable.
-// Warning 6328: (304-328): CHC: Assertion violation happens here.\nCounterexample:\na = false, x = 3, d = 0\n\n\n\nTransaction trace:\nconstructor()\nState: a = false, x = 0, d = 0\nf()
+// Warning 6328: (304-328): CHC: Assertion violation happens here.\nCounterexample:\na = false, x = 3, d = 0\n\n = 0\n\nTransaction trace:\nconstructor()\nState: a = false, x = 0, d = 0\ng()
```

Comment:
I think the CEX has now changed from calling `f` directly to calling `g` and then `f` through the external `d.d()`.
But we need full trace report to judge the correctness of the CEX.